_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,227 @@
+---
+id: 36b590
+name: Error message describes invalid form field value
+rule_type: atomic
+description: |
+  This rule checks that text error messages provided when the user completes a form field with invalid values or using an invalid format identify the cause of the error or how to fix the error.
+accessibility_requirements:
+  wcag20:3.3.1: # Error Identification (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Carlos Duarte
+    - João Vicente
+    - Wilco Fiers
+---
+
+## Applicability
+
+The rule applies to each HTML element that has one of the following [semantic roles][semantic role]: `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` or `textbox`.
+
+**Note**: The list of applicable [semantic roles][semantic role] is derived by taking all the [ARIA 1.1][] roles that:
+
+- inherit from the [abstract][] `input` or `select` role, and
+- do not have a [required context][] role that itself inherits from one of those roles.
+
+## Expectation 1
+
+For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] allows the identification of the related test target, through [text][], or [non-text content][], or through [presentation][].
+
+**Note**: This rule does not test [form field error indicators][form field error indicator] shown on a different page than the one of the test target.
+
+**Note**: A single [form field error indicator][] can be related to multiple test targets. For example, an error message at the top of a form can list all the form fields that are required and are empty.
+
+**Note**: A single test target can be related to multiple [form field error indicators][form field error indicator]. For example, a text field can have a red border around it, an error icon adjacent to it, an error message below it, and another error message at the top of the form. All of these are error indicators for the same form field.
+
+## Expectation 2
+
+For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes, in [text][] that is [visible][], the cause of the error or how to resolve it.
+
+## Expectation 3
+
+For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes:
```

Comment:
Both expectation 2 and 3 has the same outcome derived from a different set of values, so may be they should be just one expectation. Also the conjugation of the sentences is different with each of the expectations. Can we re-work/ rephrase this?

Reply:
Isn't what you say about expectation 2 and 3, true about all expectations? :-)
I have rephrased expectation 2 to be like expectation 3.